Apps/WebClient/src/ClientApp/src/services/winstonLogger.ts

```diff
@@ -1,14 +1,18 @@
 import { Logger, createLogger, format, transports } from "winston";
 import { ILogger } from "@/services/interfaces";
 import { injectable } from "inversify";
+import { ExternalConfiguration } from "@/models/configData";
 
 @injectable()
 export class WinstonLogger implements ILogger {
     private logger!: Logger;
 
-    public initialize(): void {
+    public initialize(config: ExternalConfiguration): void {
```

Comment:
Instead of passing the entire config, you can pass only the WebClientConfiguration object which contains the data you need.